Atapiave/QA_team
# FIX: Adds the hability to ignore the OS Previous Phone Number Selection

## Any background context you want to provide?

In some devices there is a prompt to select a phone number.

## What does this PR do?

Adds the hability to dismiss the Phone selection prompt. Also the DEFAULT_TIMEOUT variable was modified from 20 secconds to 5, so the scripts are being executed faster when encountering the scenario in which the phone prompts are not being shown.

## Where should the reviewer start?

From this file:
* Behave/screens/login/login_screen.py


## How should this be tested?

By creating an object of the LoginScreen class and using the login method. Or by running the existing Login test as follows:

1.- Clone this branch
2.- Create an .env file by creating a copy of the .env-sample
3.- Fill the .env file with the details of a user that exists in the DEV environment and other required details like:
OTP_API_URL = 'http://public-cluster-fargate-superapp-259682327.us-east-1.elb.amazonaws.com/superapp/qa/otp/v1/'
4.- Add an android baz-dev.apk inside the Behave/APP/ folder
5.- Open a Terminal window (Could be inside of the IDE you are using)
6.- Get inside the Behave folder
7.- Run the following command:
paver run_feature android <YourDeviceID> login 
8.- Check in your physical phone that the login process is being executed.


## What are the relevant tickets?

TBD

## Questions :question:
#### Checklist
